spec/validations/secure_settings_spec.rb

```diff
@@ -4,12 +4,22 @@
 
 describe ZendeskAppsSupport::Validations::SecureSettings do
   let(:package) { double('Package', warnings: []) }
-  let(:secured_param)  { ZendeskAppsSupport::Manifest::Parameter.new('name' => 'my_token', 'secure' => true) }
-  let(:insecure_param) { ZendeskAppsSupport::Manifest::Parameter.new('name' => 'my_token') }
+  let(:secured_param_no_scopes) { ZendeskAppsSupport::Manifest::Parameter.new('name' => 'my_token', 'secure' => true) }
+  let(:secured_param_oauth_no_scopes) { ZendeskAppsSupport::Manifest::Parameter.new('name' => 'oauth_token', 'secure' => true) }
+  let(:secured_param_with_scopes) do
+    ZendeskAppsSupport::Manifest::Parameter.new('name' => 'token_with_scopes', 'secure' => true, 'scopes' => ['header'])
+  end
+  let(:secured_param_with_nil_scopes) do
+    ZendeskAppsSupport::Manifest::Parameter.new('name' => 'token_with_nil_scopes', 'secure' => true, 'scopes' => nil)
+  end
+  let(:secured_param_with_empty_scopes) do
+    ZendeskAppsSupport::Manifest::Parameter.new('name' => 'token_with_empty_scopes', 'secure' => true, 'scopes' => [])
+  end
+  let(:insecure_param) { ZendeskAppsSupport::Manifest::Parameter.new('name' => 'my_insecure_token') }
```

Comment:
`let` blocks are lazy evaluated, so the variable is set to the closest definition. Local variables are preferred over `let` as they are hard to understand when the test suite grows (see https://thoughtbot.com/blog/my-issues-with-let).

Since this test is already using them, consider overriding `secured_param` and `insecure_param` in the contexts where its needed.